apache/superset
# fix(generator): more cleanup to plugin framework

### SUMMARY
Changes:
- Add "dev" mode to npm scripts for automatic rebuilding to replace linking to source files. This removes the need for having a Webpack alias.
- Clean up `README.md` and add instructions for adding the plugin to Superset instead of as an embedded `SuperChart`
- Add new docs section for creating new viz plugins

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS
1. go into the generator directory: `cd `superset-frontend/packages/generator-superset`
2. run `npm link`
3. create a new plugin directory: `cd ~/src && mkdir superset-plugin-hello-world`
4. initialize the plugin: `yo @superset-ui/superset`
5. Follow the instructions in the `README.md` file and see if you can make your plugin work

### ADDITIONAL INFORMATION
